### DISCUSSION TITLE
'Invalid JSON for request body' error despite JSON being valid

### DISCUSSION THREAD

**Discussion by ghost** [Budibase/budibase#8716] [Help]
Hello, 

I'm trying to create a new API request with the following body

API method: PATCH
API URL: https://api.airtable.com/v0/baseID/tableName/{{recordID}}

{{recordID}} is a binding I have set up on the call. 

Body
```
{
  "fields": {
    "Product Status": "Not received",
    "Barcode": "notReceived",
    "Last edited by": "{{Binding.user}}"
  }
}
```

The JSON is valid, however each time I hit the 'Send' button I keep getting the error `Query Error: Invalid JSON for request body`. 

The request body and call are as the Airtable API docs require, so I'm a bit confused. 

Anyone have any ideas?

**Comment by melohagan** [comment]
Hey @adamjdavies 

What happens if you replace the binding with a test value? The binding may possibly be adding quotes already, so maybe it should just be: `"Last edited by": {{ Binding.user }}`

Are you able to see what payload is being sent via the console/network tabs?
